spectrum_io/search_result/msfragger.py

```diff
@@ -61,10 +63,11 @@ def update_columns_for_prosit(df, tmt_labeled: str) -> pd.DataFrame:
         logger.info("Adding TMT fixed modifications")
         df["MODIFIED_SEQUENCE"] = msfragger_to_internal(
             df["modified_peptide"].to_list(),
-            fixed_mods={"C": "C[UNIMOD:4]", r"n[\d+]": f"{unimod_tag}-", "K": f"K{unimod_tag}"},
-        )
+            fixed_mods={"C": "C[UNIMOD:4]", r"n[\d+]": f"{unimod_tag}-", "K": f"K{unimod_tag}"}, stat_custom_mods=custom_stat_mods, 
```

Comment:
this won't work at the moment because what you supply into the function as custom staitc and var mods is Dict[str, Tuple[str, float]] but you need Dict[str,str]. Also, please do not change the signatures of those baseline functions <search_engine>_to_internal functions, they should all accept only one argument for static and var mods, i.e. this is where you would combine fixed_mods and your stat_custom_mods. Afaik, for tmt, the user is currently not supplying the tmt as a custom static mod, because it is supplied via tag param in the config, this is sth. that you should change in oktoberfest, i.e. remove the tag param and force people to provide the tmt unimod as a custom static modification.